releasenotes/notes/parallel-closeness-centrality-5165f62e370181eb.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    The :func:`~.closeness_centrality` method is now parallel for better performance.
```

Comment:
Do you want to mention `newman_weighted_closeness_centrality` here too since it's a separate function?

Reply:
In theory `newman_weighted_closeness_centrality` is a function that has not been released publicly yet, it will be first available in 0.17. It already has its own release note, it is like if we implemented it in parallel from the start.